app/views/homepage/index.html.erb

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  <link href='https://fonts.googleapis.com/css?family=Open+Sans:300' rel='stylesheet' type='text/css'>
+  </head>
+  <body>
+
```

Comment:
@slipnslide should we put everything above here in the layouts/application.html.erb file as opposed to this one?